pkg/network/config.go

```diff
@@ -1,4 +1,88 @@
 package network
 
-// CNIConfigDir is the path where CNI config files exist
-const CNIConfigDir = "/etc/cni/net.d"
+import (
+	"encoding/json"
+	"net"
```

Comment:
Import ordering

Reply:
i think this is correct.  both are internal golang thigns no?

Reply:
Oh, you're right